src/lightning-mocks/avatar/avatar.js

```diff
@@ -8,10 +8,9 @@ import { LightningElement, api } from 'lwc';
 
 export default class Avatar extends LightningElement {
     @api alternativeText
-	@api fallbackIconName
-	@api initials
-	@api size
-	@api src
-	@api variant
-    
+    @api fallbackIconName
+    @api initials
+    @api size
+    @api src
+    @api variant
```

Comment:
Lines 11-15 didn't change?

Reply:
Ah, I changed the script that generates these to use spaces instead of tabs :)